modified init script to put ref idx img in local ssd

SHuang-Broad opened the pull request:
rather than the default boot HDD.
Thanks to @mwalker174 for discovering this.

In terms of runtime, this didn't change runtime for the SV pipeline much.

mwalker174 commented on scripts/sv/default_init.sh:
```diff
@@ -15,5 +15,5 @@ if [[ ! "${ROLE}" == 'Master' ]]; then
       REFLOC+="/"
     fi
-    mkdir -p /reference && gsutil -m cp "$REFLOC"*.img /reference/
+    mkdir -p /mnt/1/reference && gsutil -m cp "$REFLOC"*.img /mnt/1/reference/
```
Maybe add a comment explaining /mnt/1/ like you have in runWholePipeline here, just in case this fails if the default path changes in the future.

SHuang-Broad commented:
@mwalker174 . Thanks for taking a look! I've added the requested comments and also added a check, in case the cluster was NOT created with a local SSD.

SHuang-Broad commented:
@mwalker174 Hi Mark, are you OK with the new changes? Thanks!

mwalker174 commented:
Yes looks great! Sorry for the late reply - didn't realize I was the only reviewer.
